model/event/host_event.go

```diff
@@ -110,20 +110,46 @@ func LogHostCreated(hostId string) {
 	LogHostEvent(hostId, EventHostCreated, HostEventData{})
 }
 
-func LogHostStartFinished(hostId string, successful bool) {
-	LogHostEvent(hostId, EventHostStarted, HostEventData{Successful: successful})
+// LogHostStartSucceeded logs an event indicating that the host was successfully
+// started.
+func LogHostStartSucceeded(hostID string) {
+	LogHostEvent(hostID, EventHostStarted, HostEventData{Successful: true})
 }
 
-func LogHostFallback(hostId string) {
-	LogHostEvent(hostId, EventHostFallback, HostEventData{})
+// LogHostStartSucceeded logs an event indicating that the host errored while
+// starting.
+func LogHostStartError(hostID, logs string) {
```

Comment:
I think it would also be fine to just keep LogHostStartFinished and also take in a log string, rather than having many versions of all these functions, since settings logs to the empty string isn't itself a problem, but I don't feel strongly